src/audio/chain_dma.c

```diff
@@ -243,6 +294,9 @@ static enum task_state chain_task_run(void *data)
 	case -EPIPE:
 		tr_warn(&chain_dma_tr, "chain_task_run(): dma_get_status() link xrun occurred,"
 			" ret = %u", ret);
+#if CONFIG_IPC_MAJOR_4
```

Comment:
is chain DMA actually available with IPC3? Has it been tested with it? It seems IPC4-only to me, in fact we'd want a dependency somewhere in Kconfig I guess